packages/webpack/mixins/fast-dev/mixin.core.js

```diff
@@ -0,0 +1,62 @@
+const { Mixin } = require('hops-bootstrap');
+
+function findResolved(a, b) {
+  try {
+    return a === require.resolve(b);
+  } catch {
+    return false;
+  }
+}
+
+class FastDevWebpackMixin extends Mixin {
+  configureBuild(webpackConfig, { allLoaderConfigs, jsLoaderConfig }, target) {
+    if (
+      target === 'build' ||
+      (target === 'node' && process.env.NODE_ENV === 'production')
+    ) {
+      return;
+    }
+
+    if (this.options.fastDev) {
+      process.env.__HOPS_FAST_DEV__ = 'true';
+
+      jsLoaderConfig.exclude = [/node_modules\/(?!(@xingternal|hops-.*)\/).*/];
```

Comment:
`@xingternal`? what is this? never heard of it! 😜

Reply:
Oh, you're right. This is breaking the separation and it should be handled in our internal mixins.